arangod/Network/Methods.cpp

```diff
@@ -111,6 +111,32 @@ auto prepareRequest(RestVerb type, std::string path, VPackBufferUInt8 payload,
   return req;
 }
 
+/// @brief Function to produce a response object from thin air:
+static std::unique_ptr<fuerte::Response> buildResponse(fuerte::StatusCode statusCode, Result const& res) {
+  VPackBuffer<uint8_t> buffer;
+  VPackBuilder builder(buffer);
+  {
+    VPackObjectBuilder guard(&builder);
+    auto errorNum = res.errorNumber();
+    builder.add(StaticStrings::Error, VPackValue(errorNum != TRI_ERROR_NO_ERROR));
```

Comment:
In the 3.7 back port of this PR there is comment about this Error generation here.

Please make sure it gets considered in both PRs.

Reply:
Did look at the comment, and did a change, here and over there.